common/configuration.ts

```diff
@@ -62,10 +62,13 @@ export interface ITokens {
   cbETH?: string
   STG?: string
   sUSDC?: string
+  sUSDbC?: string
```

Comment:
So what is the difference between `sUSDC` `sUSDbC` and `sgUSDC` and `sgUSDbC`? Arent they the same? if thats the case we should only keep one I think

Reply:
sUDSC is the stargate pool, sgUSDC is our wrapped version

Reply:
updated to wsgUSDC / wsgUSDbC